client/src/app/+signup/+register/register-step-terms.component.html

```diff
@@ -2,8 +2,8 @@
   <div class="form-group form-group-terms">
     <my-peertube-checkbox inputName="terms" formControlName="terms">
       <ng-template ptTemplate="label">
-        <ng-container i18n>
-          I am at least 16 years old and agree
+        <ng-container i18n> 
+          I am at least {{ minimumAge }} years old and agree 
```

Comment:
Please remove added spaces at the end of these lines

Reply:
```suggestion
          I am at least {{ minimumAge }} years old and agree
```